packages/e2e/README.md

```diff
@@ -123,55 +127,6 @@ You can override this port mapping by setting the `FILE_SERVER_PORT` environment
 
 `FILE_SERVER_PORT=9874 docker compose -p local-network-e2e up`
 
-## Load Testing
-
-Cardano services end to end load tests. Please note that you must have several services up before executing the test, to start the environment(from the root):
-
-```bash
-$ cd packages/cardano-services
```

Comment:
This has been omitted from the replaced documentation, so looks to me like we have a gap.

Reply:
Hi @rhyslbw ,

what was called `Load Testing` is now called `RabbitMQ Load Testing`. I did this change because when we introduced this test was the only load test of the project so there was no ambiguities.
Now we are introducing new load tests, so I thought now having something called `load testing` could be prone to ambiguities.
I applied to this documentation block following changes:

1. renamed
2. moved after the `Local Network` section as that is required in order to run this test.
3. removed all the parts _cut & pasted_ from the `Local Network` section (one of the targets of this PR)